src/static/css/cf-enhancements.less

```diff
@@ -972,6 +972,10 @@ textarea.input__long {
 .content__half-top-on-desk {
     .respond-to-min(800px, {
         padding-top: 20px;
+
+        &:after {
+          top: 20px!important;
```

Comment:
It's working, otherwise it gets set farther down the cascade, but MDN seems to recommend a space so I will update.

Sidenote, why do we use margin-top for sidebar and content blocks instead of using padding on the wrapper div? Seems we're constantly over-writing it and having no space should be the default.